src/components/Main/Burger/index.js

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components";
+
+const Burger = ({ingredientsOrder,totalPrice}) => {
+    return (
+        <BurgerStyled>
+            <TotalStyled> Total price {totalPrice} UAH</TotalStyled>
+            <BurgerWrapperStyled>
+                <TopBurgerStyled
+                src={require(`../../../assets/burgerItems/top_bun.png`)}
+                />
+                {!ingredientsOrder.length && (<PropositionStyled>
+                   Start adding ingredients
+                   </PropositionStyled>)}
+                    {ingredientsOrder.map((item, index) => {
+                        return (
+                            <ItemIMGStyled 
```

Comment:
Sometimes ingredients overlap the bun 
<img width="755" alt="image" src="https://user-images.githubusercontent.com/40695369/209438724-42355359-191d-48d9-aeb2-06cf6c406bed.png">